person-segmentation/src/util.ts

```diff
@@ -16,7 +16,6 @@ export function toInputTensor(input: PersonSegmentationInput) {
 export function cropAndResizeTo(
```

Comment:
Is there a particular reason not to use the Tensorflow.js implementation of crop_and_resize op? (I am thinking their native implementation will be faster?)

https://js.tensorflow.org/api/0.13.0/#image.cropAndResize